src/logger/src/metrics.rs

```diff
@@ -194,6 +194,9 @@ pub trait Metric {
     }
     /// Returns current value of the counter.
     fn count(&self) -> usize;
+    #[cfg(target_arch = "x86_64")]
+    /// Stores `value` to the current counter.
+    fn store(&self, value: usize);
```

Comment:
This is missing a unit test, and I think `test_metric` could be extended with one

Reply:
I think this is a good place for `store` too https://github.com/firecracker-microvm/firecracker/blob/4cb9b39f03f58498195995b209e9119c075df9a0/src/api_server/src/lib.rs#L112
We've made do with `add` so far because the metric is only emitted once but `store` is conceptually better.

Reply:
Done.

Reply:
I added it in the same commit to kind of justify the need for it (the snapshot context), but since it can be used in other places and it's an independent thing I'll move it though to another commit.